lib/govuk_app_config/govuk_error.rb

```diff
@@ -40,6 +40,8 @@ def self.configure
 
     Sentry.init do |sentry_config|
       config = Configuration.new(sentry_config)
+      # Avoid "Sending envelope with items ... to Sentry" logspew on stdout.
+      config.logger.level = Sentry::Logger::WARN
```

Comment:
Do you know if this is reconfiguring the logger for a whole application here? I'm wondering if config.logger is the Rails logger for the whole app.

Reply:
Looks like it might be I'm afraid: 

```
[38, 47] in /Users/kevin.dew/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/govuk_app_config-4.9.0/lib/govuk_app_config/govuk_error.rb
   38:       warn "Warning: GovukError is not configured to track Sidekiq errors, install the sentry-sidekiq gem to track them."
   39:     end
   40:
   41:     Sentry.init do |sentry_config|
   42:       config = Configuration.new(sentry_config)
   43:       require "byebug"; byebug # DEBUG @kevindew
=> 44:       yield config if block_given?
   45:     end
   46:   end
   47: end
(byebug) Rails.logger.object_id == config.logger.object_id
true
```

Reply:
Damn. Good catch, thanks! I shouldn't have trusted the docs 🙈 Stand by for bugfix release :)